lib/internal/util.js

```diff
@@ -532,7 +533,10 @@ function defineLazyProperties(target, id, keys, enumerable = true) {
       mod ??= require(id);
```

Comment:
Can I suggest moving this line inside the `if` statement below? If the getter can't be replaced, it should do as little work as possible in the common path.

Reply:
I suppose by the `if` statement you meant `if (lazyLoadedValue === undefined)`? In that case it made sense to me, otherwise that's probably not the desired behavior (we can still return the result in the getter, just can't fix up the descriptor when it's frozen).